router/pkg/routerconfig/s3/client.go

```diff
@@ -30,13 +31,29 @@ type ClientOptions struct {
 }
 
 func NewClient(endpoint string, options *ClientOptions) (routerconfig.Client, error) {
-
 	client := &Client{
 		options: options,
 	}
 
+	// The providers credential chain is used to allow multiple authentication methods.
+	providers := []credentials.Provider{
```

Comment:
For consistency, please add the same functionality to the persisted operation [client](https://github.com/wundergraph/cosmo/blob/main/router/internal/persistedoperation/s3/client.go).

Reply:
That's done now https://github.com/wundergraph/cosmo/pull/1250/commits/286c2743c65ddd4ade47636ebd58c038faf8d739